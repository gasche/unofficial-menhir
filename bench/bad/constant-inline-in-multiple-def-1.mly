%start<int> b 
%%

b: {}
%public a: {}
