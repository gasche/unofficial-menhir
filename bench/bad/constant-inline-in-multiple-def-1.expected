File "constant-inline-in-multiple-def-2.mly", line 3, characters 16-17:
File "constant-inline-in-multiple-def-1.mly", line 5, characters 8-9:
Error: not all definitions of a are marked %inline.
